### PR TITLE
Ci

### PR DESCRIPTION
**Pull request by SnakeHacker**

### CODE REVIEW

**Comment by SnakeHacker** [comment]
@iminders PTAL

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: CI
+
+on: [push, pull_request]
```
Comment (iminders) [inline .github/workflows/ci.yml]: `push, pull_request` -> `push`

Comment (SnakeHacker) [reply .github/workflows/ci.yml]: Done

Comment (SnakeHacker) [reply .github/workflows/ci.yml]: Done

**Review by iminders — APPROVED** [review]
LGTM

**Pull request merged**